src/js/toggler.js

```diff
@@ -17,6 +23,35 @@ const initToggler = () => {
         }
       })
   })
+
+  checkboxPopper.addEventListener('click', function () {
```

Comment:
Should't we use change here?
```suggestion
  checkboxPopper.addEventListener('change', function () {
```

Reply:
I'll try and check how it's behave with that change 👍